src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -137,11 +137,11 @@ public async Task DeprecateAsync(string id, IReadOnlyCollection<string> versions
         public async Task SearchPackageODataV2FromDBAsync(string id, string normalizedVersion, ITestOutputHelper logger)
         {
             var galleryEndpoint = GetGalleryServiceBaseUrl();
-            // Add 1 eq 1 to block the search hijack and execute the query against the database.
-            var urlRootAndPath = $"{galleryEndpoint}/api/v2/Packages()";
+            // Add the hijack=false query parameter to block the search hijack and execute the query against the database.
+            var urlRootAndPath = $"{galleryEndpoint}/api/v2/Packages(Id='{id}',Version='{normalizedVersion}')";
             var queryParameters = new Dictionary<string, string>()
             {
-                { "$filter", $"Id eq '{id}' and NormalizedVersion eq '{normalizedVersion}' and 1 eq 1" }
```

Comment:
Is this easily testable? If so, consider adding a test :)

Reply:
I think there is limited value in unit testing trivial parts of our end-to-end tests. They themselves are run as tests. If there is a complex test harness or some tricky code, I agree.

If you feel strongly, I'm happy to investigate it.

Reply:
Agreed, feel free to skip 👍